addons/repair/XEH_postInit.sqf

```diff
@@ -1,9 +1,13 @@
 #include "script_component.hpp"
 
+if !(EGVAR(common,settingsInitFinished)) exitWith {
+    EGVAR(common,runAtSettingsInitialized) pushBack [compileScript [__FILE__], _this];
+};
```

Comment:
May I use such code instead of moving the whole file into `CBA_settingsInitialized` frame? Also this can be used:
```suggestion
if (param [0, true]) exitWith {
    ["CBA_settingsInitialized", {false call compileScript [__FILE__]}] call CBA_fnc_addEventHandler;
};
```

Reply:
I'm not a fan of either solution, would rather just wrap everything in the EH.

Reply:
yeah, it might make the diff ugly but I think the cleanest way is to just tab everything over in the eh